components/integrations/installed_incoming_webhooks/installed_incoming_webhooks.tsx

```diff
@@ -79,6 +82,14 @@ export default class InstalledIncomingWebhooks extends React.PureComponent<Props
         return displayNameA.localeCompare(displayNameB);
     }
 
+    nextPage = () => {
+        this.setState({page: this.state.page + 1});
+    }
+
+    previousPage = () => {
+        this.setState({page: this.state.page - 1});
+    }
+
```

Comment:
there is no need for a specific handler function here, since there are no side-effects being needed